### DISCUSSION TITLE
대시보드/컬럼 이름 글자수 제한 (v)

### DISCUSSION THREAD

**Discussion by un0211** [Part3-Team15/taskify#149] [General]
<img width="960" alt="image" src="https://github.com/Part3-Team15/taskify/assets/24778465/5a0a2b8f-a38e-4272-9656-1f065f8fb309">
<img width="1191" alt="image" src="https://github.com/Part3-Team15/taskify/assets/24778465/6b7f3c99-d352-4737-8132-caaba096fce3">
<img width="1183" alt="image" src="https://github.com/Part3-Team15/taskify/assets/24778465/1a5f83b0-5338-41e5-b9e3-35c316aee28c">

API에서 글자제한을 두지 않고 받는 것 같은데, 디자인이 이상해지는 문제가 있습니다.
- 디자인 개선
  - ... 으로 가렸다가 버튼 누르면 다 보게 할 수 있을 것 같습니다.
- 글자수 제한 (선호)
  - 대시보드 / 컬럼 이름 모두 글자수 10자 정도면 충분하지 않을까 생각합니다.

**Comment by wjsdncl** [comment]
디자인은 ... 으로 수정하겠습니다
글자수는 10자 충분한거 같습니다! 👍
